### PR TITLE
Punctuation and typo fixes

### PR DESCRIPTION
**Pull request by zakhar-kogan**
Fixed typos and commas + scanned README with Grammarly 🙌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Our goal is to not reinvent the wheel where appropriate.
 
 PyTorch is designed to be intuitive, linear in thought, and easy to use.
 When you execute a line of code, it gets executed. There isn't an asynchronous view of the world.
-When you drop into a debugger or receive error messages and stack traces, understanding them is straightforward.
+Wnderstanding them is straightforward when you drop into a debugger or receive error messages and stack traces.
```
Comment (kit1980) [inline README.md:111]: "Wnderstanding"

**Comment by kit1980** [comment]
Many of these look somewhat controversial to me, and at least one new issue was introduced as I commented.

I know there are plenty of actual non-controversial typos in the codebase (see https://github.com/pytorch/pytorch/pull/83056 for example).

**Comment by kit1980** [comment]
Also one of the changes was already proposed in https://github.com/pytorch/pytorch/pull/85075 and there is some discussion there.

**Comment by zaharkogan** [comment]
> Also one of the changes was already proposed in #85075 and there is some discussion there.

Got that -> judging from the way discussion went, should I close this PR?
On non-controversial checks: will spot them if I see those 'in bulk'.

**Pull request closed without merging**